package.json

```diff
@@ -27,19 +29,37 @@
     "parser",
     "swift"
   ],
+  "files": [
+    "grammar.js",
+    "binding.gyp",
+    "prebuilds/**",
+    "bindings/node/*",
+    "queries/*",
+    "src/**"
+  ],
   "author": "Alex Pinkus <alex.pinkus@gmail.com>",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/alex-pinkus/tree-sitter-swift/issues"
   },
   "homepage": "https://github.com/alex-pinkus/tree-sitter-swift#readme",
   "dependencies": {
-    "nan": "^2.18.0",
-    "tree-sitter-cli": "=0.20.6",
```

Comment:
This was previously pinned to `0.20.6` which is why the latest arm64 updates weren't working. I took a look at the commit for why this was pinned in the first place but wasn't completely sure what it was referencing.